app/1.0/blog/2016-09-09-polymer-2.0.md

```diff
@@ -0,0 +1,95 @@
+---
+title:  "Polymer 2.0 Preview"
+---
+
+Today we pushed a [`2.0-preview`](https://github.com/polymer/polymer/tree/2.0-preview) branch to the Polymer repository, where we’re working on the next major version of the library.
+
+Polymer 2.0 is a minimally-breaking change from the current library, with a few very precise goals:
+
+### Take advantage of native [v1 Shadow DOM](https://developers.google.com/web/fundamentals/primers/shadowdom/) and [v1 Custom Elements](https://developers.google.com/web/fundamentals/primers/customelements/) API’s
+
+The primary goal of the Polymer 2.0 release is to take advantage of native, cross-browser support for Web Components.
+
+   Polymer 1.x is built on top of the so-called "v0" Web Components specs, which are supported natively only in Google Chrome; using Polymer in other browsers has always required the use of polyfills.
+
+   Beginning this fall, multiple browsers will be shipping native implementations of the new "v1" specs for Shadow DOM and Custom Elements, yielding better web components performance and reducing the need for polyfills.
```

Comment:
May be worth alluding to what browsers we're referring to here for folks that haven't been following along.